### DISCUSSION TITLE
Help me to debug and develop

### DISCUSSION THREAD

**Discussion by lyducanh** [evershopcommerce/evershop#567] [Q&A]
hello,I installed project with "npm run build" then "npm run start" how can i debug this app if all code already built and placed in /.evershop/build/* folder

**Comment by treoden** [comment]
Hi,

You can run the dev mode. `npm run dev`

**Comment by vcfvct** [comment]
HI @treoden , thanks for the awesome framework. 
For the backend nodejs code, is there any way we can run in debug session, set breakpoint in the `node_modules` and debug? :)
